pkg/controller/linstornodeset/linstornodeset_controller.go

```diff
@@ -69,6 +68,9 @@ var log = logrus.WithFields(logrus.Fields{
 // to them.
 const linstorNodeFinalizer = "finalizer.linstor-node.linbit.com"
 
+// Default value for AutomaticStorageSetupType. If set, no automatic setup of storage devices happens.
+const defaultAutomaticStorageSetupType = "None"
```

Comment:
I would call this `automaticStorageTypeNone` because isn't just the default; it has a special meaning when this value is used.